lib/web_ui/lib/src/engine/canvaskit/vertices.dart

```diff
@@ -83,10 +84,10 @@ class CkVertices extends ManagedSkiaObject<SkVertices> implements ui.Vertices {
   );
 
   final SkVertexMode _mode;
-  final Float32List _positions;
-  final Float32List? _textureCoordinates;
-  final Uint32List? _colors;
-  final Uint16List? _indices;
+  late Float32List _positions;
```

Comment:
Is it actually helpful to null out fields like this? The typed data isn't external so the VM already knows the size, and the only way to clean it up is with a GC

Reply:
Ahh fair. I'll revert this part.

Reply:
We could add a new vertices constructor which allocates external typed data with fixed sizes in the format that impeller needs, then change the API so that the dart code writes into that external typed data. Would avoid some copies and could be disposed